packages/components/src/components/Flyout/Flyout.stories.tsx

```diff
@@ -0,0 +1,193 @@
+                </Flyout.Trigger>
+                <Flyout.Content {...args}>
+                    <Flyout.Header>Header</Flyout.Header>
+                    <Flyout.Body>I am a flyout</Flyout.Body>
+                </Flyout.Content>
+            </Flyout.Root>
+        );
+    },
+};
+
+export const WithCloseButton: Story = {
+    name: 'With Close Button',
+    render: ({ ...args }) => {
+        return (
+            <Flyout.Root>
+                <Flyout.Trigger>
+                    <Button>Click Me</Button>
+                </Flyout.Trigger>
+                <Flyout.Content {...args}>
+                    <Flyout.Header showCloseButton>Header</Flyout.Header>
+                    <Flyout.Body>I am a flyout</Flyout.Body>
+                </Flyout.Content>
+            </Flyout.Root>
+        );
+    },
+};
+
+export const WithComfortablePadding: Story = {
+    name: 'With Comfortable Padding',
+    args: {
+        padding: 'comfortable',
+    },
+};
+
+export const WithSpaciousPadding: Story = {
+    name: 'With Spacious Padding',
+    args: {
+        padding: 'spacious',
+    },
+};
+
+export const WithCustomContent: Story = {
+    name: 'With Custom Conent',
+    render: ({ ...args }) => {
+        return (
+            <Flyout.Root>
+                <Flyout.Trigger>
+                    <Button>Click Me</Button>
+                </Flyout.Trigger>
+                <Flyout.Content {...args}>
+                    <div className="tw-bg-box-neutral tw-p-10">This is FlyoutContent</div>
+                </Flyout.Content>
+            </Flyout.Root>
+        );
+    },
+};
+
+export const RightSide: Story = {
+    name: 'Right Side',
+    args: {
+        side: 'right',
+    },
+};
+
+export const AlignCenter: Story = {
+    name: 'Align Center',
+    args: {
+        side: 'right',
+        align: 'center',
+    },
+};
+
+export const AlignEnd: Story = {
+    name: 'Align End',
+    args: {
+        side: 'right',
+        align: 'end',
+    },
+};
+
+export const LongContent: Story = {
+    name: 'Long Content',
+    args: {
+        side: 'bottom',
+        children:
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.',
+    },
+};
+
+export const CustomMaxWidth: Story = {
+    name: 'Custom Max Width',
+    args: {
+        maxWidth: '200px',
+        side: 'bottom',
+        children:
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.',
+    },
+};
```

Comment:
Please add a view for mobile 
https://storybook.js.org/docs/essentials/viewport#configuring-per-component-or-story

Reply:
added